user-guide/Getting_started/Getting_Started.md

```diff
@@ -7,7 +7,7 @@ description: Create a DataMiner System, and learn how to interact with DataMiner
 
 <div class="row"> 
   <div class="column">
-    <a href="/user-guide/Getting_started/Creating_a_DataMiner_System.html" title="Creating a DMS" target="_self"><img src="~/user-guide/images/Creating_DMS.svg" style="width:100%"></a>
+    <a href="/user-guide/Getting_started/Deploying_a_DataMiner_System.html" title="Creating a DMS" target="_self"><img src="~/user-guide/images/Creating_DMS.svg" style="width:100%"></a>
```

Comment:
@EdithLansens, when you have a moment, could you replace this SVG so it says "deploying" instead of "creating"?